packages/cli/tests/unit/agentInstall/commandRunner.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { randomBytes } from 'crypto';
+import { run } from '../../../src/cmds/agentInstaller/commandRunner';
+import CommandStruct from '../../../src/cmds/agentInstaller/commandStruct';
+
+describe('CommandRunner', () => {
+  describe('run', () => {
+    it('raises an error when a command is not found', async () => {
+      const unknownCmd = randomBytes(8).toString('hex');
+      try {
+        await run(new CommandStruct(unknownCmd, [], process.cwd()));
+      } catch (err) {
+        expect(err.message).toMatch(`${unknownCmd} was not found`);
+        return;
+      }
+      fail('Expected an error to be thrown');
```

Comment:
You can clean up the end of this test a bit by putting an `expect.assertions(1)` at the beginning of the function: https://jestjs.io/docs/expect#expectassertionsnumber and getting rid of the `return` and `fail`.

Personally, I prefer Jest's `toThrow` or `rejects` assertions for testing error behavior, but that's mostly a style thing.